### DISCUSSION TITLE
How do I move a flux-system from one folder to another (gotk-sync.yaml)

### DISCUSSION THREAD

**Discussion by lenaxia** [fluxcd/flux2#3147] [Q&A]
I want to move my flux-system kustomization from one folder to another (in the file `flux-system/gotk-sync.yaml`)

```apiVersion: kustomize.toolkit.fluxcd.io/v1beta2
kind: Kustomization
metadata:
  name: flux-system
  namespace: flux-system
spec:
  path: ./clusters/dev => ./clusters/prod
  sourceRef:
    kind: GitRepository
    name: flux-system
```

However at the top of the manifest file it says `# This manifest was generated by flux. DO NOT EDIT.`, Will there be any problems if I make this change? 

Or what's the best way to do this? My `gotk-sync.yaml` is in `./clusters/dev/flux-system/gotk-sync.yaml`, will this cause any complications?

**Comment by stefanprodan** [comment]
Are you also deleting your dev cluster? How did you installed Flux on the 2nd cluster? Normally you would run bootstrap for the 2nd cluster using a different path, see the example repo please https://github.com/fluxcd/flux2-kustomize-helm-example
